packages/rspack/src/config/zod/builtins.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from "zod";
+
+export function builtins() {
+	// TODO(hyf0): need to enable strict mode sometimes
```

Comment:
Can you be more clear on when "sometimes" is? These kind of todo comments tend to stay for ever, and we don't know when to remove these.